worldedit-bukkit/src/test/java/com/sk89q/worldedit/bukkit/BukkitWorldTest.java

```diff
@@ -28,6 +30,7 @@
 
     @Test
     public void testTreeTypeMapping() {
+        Bukkit.setServer(new StubServer());
```

Comment:
Surround Fawe additions in upstream classes with our annotations, see https://github.com/IntellectualSites/FastAsyncWorldEdit/blob/main/CONTRIBUTING.md#example